src/commands/util/maven/setup.ts

```diff
@@ -0,0 +1,69 @@
+import {BaseCommand} from '../../../base'
+import {flags} from '@oclif/command'
+import {randomBytes} from 'crypto'
+import {homedir} from 'os'
+import * as path from 'path'
+import * as fs from 'fs'
+import * as inquirer from 'inquirer'
+import * as readline from 'readline'
+
+const prompt = inquirer.createPromptModule()
+
+export default class UtilMavenSetup extends BaseCommand {
```

Comment:
Couple things pop to mind.

- This overwrites the settings so would blow away any existing user settings. It would need to at least back it up. Best case would be that it adds the missing settings to an existing user settings, but, that would make this much more complicated.
- Force does nothing
- No description
- The paths (particularly the logged ones) aren't all OS independent
- It duplicates existing tool functionality
- Password argument should not be used with maven. It should be input when prompted.

On balance, I'm not sure adding code someone needs to maintain (and add unit tests for) is better than a simple confluence page.